fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/Client.java

```diff
@@ -180,16 +180,14 @@ public final class Client extends AbstractPersistableCustom {
     @Column(name = "submittedon_date", nullable = true)
     private LocalDate submittedOnDate;
 
-    @ManyToOne(optional = true, fetch = FetchType.LAZY)
-    @JoinColumn(name = "submittedon_userid", nullable = true)
-    private AppUser submittedBy;
+    // Deprecated since common Auditable fields were introduced. Columns and data left untouched to help migration.
 
-    @Column(name = "updated_on", nullable = true)
-    private LocalDate updatedOnDate;
+    // @Column(name = "updated_on", nullable = true)
```

Comment:
Can we use multi-line comment here? Sorry, I know it's a minor thing but it groups these lines well together and nobody is going to confuse it with actual comments that could be deleted. Thanks.